src/main/java/com/fasterxml/jackson/core/io/NumberOutput.java

```diff
@@ -245,6 +245,10 @@ public static int outputLong(long v, byte[] b, int off)
         return _outputFullBillion((int) v, b, off);
     }
 
+    static int divBy1000(int number) {
```

Comment:
could mark this as final? likewise with the four-fifths method

Reply:
Of course I can, doesn't they supposed as being effectively final and thus treated as "real" final?

Reply:
Yes, it is tiny improvement, but as I saw the same principle (division by multiply and add) used in the project, so I expected, that this changes will be welcomed. If you look to my fork of this repo, I have prepared several PRs of this kind, so it will be convenient to know beforehand if I even should make another proposals like this not to waste our time.

Reply:
I'll let @cowtowncoder decide. For me, this sort of enhancement is difficult to maintain. If it is absolutely much faster then maybe this is worth it. We would need benchmarks.

Reply:
I can prepare benchmarks also, but as the improvement are tiny, they need to be focused on particular actions to be visible, but as you wish - I will look at _jackson-benchmarks_ and will try to find some convincing benchmarks or will try to create some for all my stuff. Then I filter my PRs to only ones, whose are "valuable".